src/components/PieChart/index.tsx

```diff
@@ -117,7 +117,7 @@ const PieChart: VoidFunctionComponent<PieChartProps> = ({
           defs={[
             {
               background: 'inherit',
-              color: colorsDeprecated.white,
+              color: colors.neutral.textWeakHover,
```

Comment:
```suggestion
              color: colors.neutral.textStrong,
```

With strong background context text should be white in same context and will fix this difference:

![Screenshot 2021-12-14 at 12 29 13](https://user-images.githubusercontent.com/15812968/145990138-5f07daab-9759-4dca-94b2-1119abdc7344.png)
![Screenshot 2021-12-14 at 12 29 18](https://user-images.githubusercontent.com/15812968/145990141-382caad1-7727-4944-baf4-2f37a2ff4b01.png)

Reply:
oops, done